docs/tutorials/plotting/pairplot_with_gmm.ipynb

```diff
@@ -0,0 +1,151 @@
+    "\n",
+    "Thus, the first 50 vertices belong to block 1, the second 50 vertices belong to block 2, and the last 50 vertices belong to block 3."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.simulations import sbm\n",
+    "\n",
+    "n_communities = [50, 50, 50]\n",
+    "p = [[0.5, 0.1, 0.05], \n",
+    "     [0.1, 0.4, 0.15], \n",
+    "     [0.05, 0.15, 0.3],]\n",
+    "\n",
+    "np.random.seed(2)\n",
+    "A = sbm(n_communities, p)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Embed using adjacency spectral embedding to obtain lower dimensional representation of the graph\n",
+    "\n",
+    "The embedding dimension is automatically chosen. It should embed to 3 dimensions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.embed import AdjacencySpectralEmbed\n",
+    "\n",
+    "ase = AdjacencySpectralEmbed()\n",
+    "X = ase.fit_transform(A)\n",
+    "\n",
+    "print(X.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use pairplot_with_gmm to plot the embedded data with ellipses\n",
+    "\n",
+    "First we generate labels that correspond to blocks. We pass the labels along with the data for to create a pairplot with the ellipses estimated by the GMM over each compnonent of the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.plot import pairplot_with_gmm\n",
+    "from sklearn.mixture import GaussianMixture\n",
+    "gmm = GaussianMixture(n_components=3, covariance_type='full').fit(X)\n",
+    "labels = ['Block 1'] * 50 + ['Block 2'] * 50 + ['Block 3'] * 50\n",
+    "cluster_palette = {0: 'red', 1:'blue', 2: 'green'}\n",
+    "label_palette = {\"Block 1\": 'red', 'Block 2':'blue', 'Block 3': 'green'}\n",
+    "pairplot_with_gmm(X, gmm, labels = None, cluster_palette = None,  label_palette = None)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to label your ellipses a specific color you can do the following"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.plot import pairplot_with_gmm\n",
+    "from sklearn.mixture import GaussianMixture\n",
+    "gmm = GaussianMixture(n_components=3, covariance_type='full').fit(X)\n",
+    "labels = ['Block 1'] * 50 + ['Block 2'] * 50 + ['Block 3'] * 50\n",
+    "cluster_palette = {0: 'red', 1:'blue', 2: 'green'}\n",
+    "label_palette = {\"Block 1\": 'red', 'Block 2':'blue', 'Block 3': 'green'}\n",
+    "pairplot_with_gmm(X, gmm, labels = labels, cluster_palette = cluster_palette,  label_palette = label_palette)"
+   ]
+  }
+ ],
```

Comment:
I'd indicate in the title that the colors are intentionally different in the last cell. Also maybe the ellipse colors should be closer to the data colors (just not the same as the default)? something like light red instead of red? tradeoff between prettiness of plot and clarity of what you're trying to show

Reply:
The ellipse and data colors are the same. I am confused about what I should be changing. Will make clear that the colors should not match in the notebook title.

Reply:
red matches with blue
i'm saying maybe red should match with a lighter red or something

My first reaction to the last plot wasn't "oh cool, this is a way to change the colors", it was "oh crap, these colors seem like they're wrong", and then I had to go back and see from the text that they were supposed to be different from the data points

Reply:
@loftusa I think then I will just make it clear in the writing that it should be the case.

Reply:
fix https://github.com/microsoft/graspologic/pull/541/commits/ed819eead52eb3bf208308105c2ad5924a31e2f0